fs-repo-12-to-13/atomicfile/atomicfile.go

```diff
@@ -0,0 +1,59 @@
+// Package atomicfile provides the ability to write a file with an eventual
```

Comment:
Should this file be on `tools` package?

Reply:
Probably, this doesn't need to be done now given that would be function-less code change.
It would help making future migrations cleaner.